CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 3.21.3)
```

Comment:
We should definitely lower the CMake requirement: Ubuntu 18 for example, ships with CMake 3.10.2

Reply:
I agree with the LTS support. I double-checked, and we should be ok with 3.10.